src/tarsnapaccount.cpp

```diff
@@ -7,6 +7,7 @@
 #include <QNetworkRequest>
 #include <QSettings>
 #include <QTableWidget>
+#include <QHeaderView>
```

Comment:
Can you move the new line atop of QMessageBox, to maintain alphabetical sort order for header includes? :)